pkg/cli/admin/inspect/inspect.go

```diff
@@ -100,6 +102,8 @@ func NewCmdInspect(streams genericclioptions.IOStreams, parentCommandPath string
 
 	cmd.Flags().StringVar(&o.destDir, "dest-dir", o.destDir, "Root directory used for storing all gathered cluster operator data. Defaults to $(PWD)/inspect.local.<rand>")
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
+	cmd.Flags().Int64Var(&o.sinceSeconds, "sinceseconds", o.sinceSeconds, "An RFC3339 timestamp from which to show logs. If this value precedes the time a pod was started, only logs since the pod start will be returned.")
+	//cmd.Flags().DurationVar(&o.sinceTime, "sincetime", o.sinceTime, "Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of sinceTime or sinceseconds may be used.")
```

Comment:
If you want to support both, use the same naming as in `oc logs`.

Reply:
This still holds.
/hold

Reply:
I am confused for this. I am not sure how to input variable for meta.Time as there is no flag for this. Can you help me with this.

Reply:
Check https://github.com/openshift/oc/blob/master/vendor/k8s.io/kubectl/pkg/cmd/logs/logs.go for how to wire and use them.